src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] I will strongly recommend you don't disable your linters because linters help to diagnose and fix technical issues—e.g. code smells—in the code and also help you write good and readable code 👍🏽 

You can consider adding initializing a constructor and a state because the class-based components (or stateful component )require a state, hence why you are getting the linter errors 😄 

```
class Calculator extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
...
```